client/components/plans/plans-compare/index.jsx

```diff
@@ -96,6 +99,39 @@ var PlansCompare = React.createClass( {
 		}, this );
 	},
 
+	showFreeTrialException: function() {
+		const hasTrial = this.props.sites ? this.props.sites.getSelectedSite().plan.free_trial : false,
```

Comment:
If `canStartTrial` returns false for sites that are currently on the free trial, I don't believe we need this condition.